feat: add cloud secrets manager docs (#537)
kestra-io/docs: content/docs/09.administrator-guide/01.configuration/03.enterprise-edition/secrets/azure-key-manager/index.md

---
title: Azure Key Vault configuration
---

This integration will store secrets in [Azure Key Vault](https://azure.microsoft.com/products/key-vault/). Kestra authentications must have `"Get", "List", "Set", "Delete", "Recover", "Backup", "Restore", "Purge"` permissions.

## Authentication

> You can handle authentication to AZURE Cloud in multiple ways:

Reviewer: ```suggestion
You can authenticate the Secrets Manager with Azure Cloud in multiple ways:
```

- Using a user or a service principal secret by setting properties `clientId` and `clientSecret`.

> - If none is set, the default authentication will be used like for AWS own cli.

Reviewer: ```suggestion
- If none is set, the default authentication will be used e.g. in a similar way as Azure CLI does it.
```
